std/base64.d

```diff
@@ -53,7 +53,7 @@
  * Authors:   Masahiro Nakagawa, Daniel Murphy (Single value Encoder and Decoder)
  * Source:    $(PHOBOSSRC std/_base64.d)
  * Macros:
- *      LREF2=<a href="#$1">$(D $2)</a>
+ *      LREF2=<a href="#$1">`$2`</a>
```

Comment:
I think this is one of the culprits - see [here](http://dtest.dlang.io/diff/website-7a608bf8457130bb33dae89311a1dcdebf01bc3b-3bcdcebf6c39ba84e8fd3f5faa3064a7/website-7a608bf8457130bb33dae89311a1dcdebf01bc3b-74648b88f0af419c3de1b2b2d2d6e5c6/web/phobos-prerelease/std_base64.html):
```diff
-  <a href="#.Base64Impl.Encoder.empty"><span class="d_inlinecode donthyphenate notranslate">empty</span></a> returns <span class="d_inlinecode donthyphenate notranslate"><code class="ddoc_keyword">true</code></span>.</div></div>
+  <a href="#.Base64Impl.Encoder.empty">`empty`</a> returns <span class="d_inlinecode donthyphenate notranslate">true</span>.</div></div>
```